bolt/lib/Profile/BoltAddressTranslation.cpp

```diff
@@ -36,9 +37,12 @@ void BoltAddressTranslation::writeEntriesForBB(MapTy &Map,
   if (BBInputOffset == BinaryBasicBlock::INVALID_OFFSET)
     return;
 
-  LLVM_DEBUG(dbgs() << "BB " << BB.getName() << "\n");
-  LLVM_DEBUG(dbgs() << "  Key: " << Twine::utohexstr(BBOutputOffset)
-                    << " Val: " << Twine::utohexstr(BBInputOffset) << "\n");
+  LLVM_DEBUG(dbgs() << "BB " << BB.getName() << "\n"
```

Comment:
So you're not using the basic block hash anywhere, other than printing it to debug output?

Reply:
Not in this diff. Hashes will be output to YAML in follow up diffs starting with https://github.com/llvm/llvm-project/pull/76910.